src/logging/logger.ts

```diff
@@ -112,38 +173,108 @@ export class Logger implements ZLUX.Logger {
     return configuredLevel >= level;
   };
 
+  private static createPrependingStrings(prependLevel?: boolean,
+                                         prependProcess?: boolean,
+                                         prependUser?: boolean): string[] {
+    let formatting = '';
+    if (prependProcess) {
+      formatting += Logger.processString;
+    }
+    if (prependUser && Logger.username) {
+      formatting += `${Logger.username} `;
+    } else if (prependUser && Logger.euid) {
+      formatting += `${Logger.euid} `;
+    }
+    if (prependLevel) {
+        return [
+          `${formatting}${LogLevel[0]} `,
+          `${formatting}${LogLevel[1]} `,
+          `${formatting}${LogLevel[2]} `,
+          `${formatting}${LogLevel[3]} `,
+          `${formatting}${LogLevel[4]} `,
+          `${formatting}${LogLevel[5]} `,
+        ];
+    } else {
+      return [
+        `${formatting} `,
+        `${formatting} `,
+        `${formatting} `,
+        `${formatting} `,
+        `${formatting} `,
+        `${formatting} `,
+      ];
+    }    
+  }
+  
   private consoleLogInternal(componentName:string,
-                           minimumLevel:LogLevel,
-                           prependDate?:boolean,
-                           prependName?:boolean,
-                           prependLevel?:boolean,
-                           ...loggableItems:any[]):void {
-    var formatting = '[';
+                             minimumLevel:LogLevel,
+                             prependingString:string,
+                             prependDate?:boolean,
+                             prependName?:boolean,
+                             ...loggableItems:any[]):void {
+    var formatting = '';
     if (prependDate) {
       var d = new Date();
-      var msOffset = d.getTimezoneOffset()*60000;
-      d.setTime(d.getTime()-msOffset);
+      d.setTime(d.getTime()-Logger.offsetMs);
       var dateString = d.toISOString();
       dateString = dateString.substring(0,dateString.length-1).replace('T',' ');
-      formatting += dateString+ ' ';
+      formatting += `${dateString} `;
     }
-    if (prependName) {
-      formatting += componentName+ ' ';
+    formatting+=prependingString;
+    //Logger.os only present in node
+    //Inspired from https://stackoverflow.com/questions/16697791/nodejs-get-filename-of-caller-function
+    //API def: https://v8.dev/docs/stack-trace-api
+    if (prependName && Logger.os) {
```

Comment:
Why not add a runtime function to switch on chrome mode if you are chrome, just in case people deliberately want this